src/content/contributing/release-process/_index.en.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Release process"
+date: 2020-03-18T16:03:26+01:00
+weight: 10
+---
+
+
+In this section of the documentation we explain the necessary steps to make a submariner release,
+it's assumed that you are familiar with the submariner project and the different repositories.
+
+
+# Step 1: release submariner (core)
+
+
+Assuming that you have an existing submariner git directory, and that you are making a release based in master.
+
+```bash
+cd submariner
+git remote add upstream ssh://git@github.com/submariner-io/submariner
+git fetch -a -v -t
+git checkout remotes/upstream/master -B master
+git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
+git push upstream v0.2.0-rc0
+```
+
+At this point the release should appear here, a github release can be created.
+
+> https://github.com/submariner-io/submariner/tags
+
+A build should start:
+
+> https://travis-ci.com/github/submariner-io/submariner/branches
+
+Look under the "Active branches" section for v0.2.0-rc0 , and monitor that the build
+completes successfully, specially the Deploy section, where the images are pushed to quay.io, those should eventually appear here after the job has finished:
```

Comment:
```suggestion
Verify that the build successfully completes as indicated by a green checkmark at the right . At this point the images are pushed to quay.io.
```
I took out the part about how the images should appear in the Deploy section. Looking at the links, I don't see a Deploy section.

Reply:
There should be at the end, I will take the suggestion any way as it's not super important.

Reply:
> There should be at the end

You mean a Deploy section? At the very end where? I look at https://travis-ci.com/github/submariner-io/submariner/branches and don't see such a section. Am I missing something?